server/base/src/main/java/org/apache/accumulo/server/iterators/TabletIteratorEnvironment.java

```diff
@@ -79,6 +80,30 @@ public TabletIteratorEnvironment(ServerContext context, IteratorScope scope,
     this.topLevelIterators = new ArrayList<>();
   }
 
+  public TabletIteratorEnvironment(ServerContext context, IteratorScope scope,
```

Comment:
What is going on w/ this new constructor?  Its only used in a test, what is preventing the test from using the new builder?

Reply:
which test?

Reply:
ah, I think it's because I was trying to get all of the server side code to use server-side iterator environment objects vs having all the server side tests using the new ClientIteratorEnvironment.